### ISSUE TITLE
AtkinsonBoore2006Modified2011 moved to boore_atkinson_2008.py in oq-engine

### ISSUE THREAD

**Issue by raychani** [usgs/shakemap#1148]
Please describe your issue here:

few days ago, the oq-engine updated AtkinsonBoore2006Modified2011 class.
the class moved to boore_atkinson_2008.py from atkinson_boore_2006

so following shakelib/utils/utils.py code may need to change
from
```
from openquake.hazardlib.gsim.atkinson_boore_2006 import (
    AtkinsonBoore2006Modified2011)
```
to
```
from openquake.hazardlib.gsim.boore_atkinson_2008 import (
    AtkinsonBoore2006Modified2011)
```
or
```
try:
    from openquake.hazardlib.gsim.atkinson_boore_2006 import (
        AtkinsonBoore2006Modified2011)
except ModuleNotFoundError as ex:
    from openquake.hazardlib.gsim.boore_atkinson_2008 import (
        AtkinsonBoore2006Modified2011)
```

---
If you are reporting a problem with running the code, please report the outputs from the following commands after cloning this reponsitory and running `install.sh`:
    - `python setup.py version`
    - `uname -v`
    - `conda --version`

**Comment by cbworden** [comment]
Thanks for bringing this to our attention. It will require a couple of other fixes as well, but I'll get to it shortly.

**Comment by cbworden** [comment]
This has been fixed at OpenQuake and is fixed in ShakeMap release 4.0.5 as well as what's currently in the repo (which isn't guaranteed to work since OpenQuake is currently a moving target.)
